mw-webapp/src/component/image/Image.tsx

```diff
@@ -25,21 +35,66 @@ interface ImageProps {
    * Data attribute for cypress testing
    */
   dataCy?: string;
+
+  /**
+   * Enlarge Image
```

Comment:
Describe comments in right way. You can see example in other components

Reply:
Done